airgun/settings.py

```diff
@@ -69,6 +69,7 @@ def __init__(self):
         self.satellite = SatelliteSettings()
         self.selenium = SeleniumSettings()
         self.webdriver_desired_capabilities = WebdriverCapabilitiesSettings()
+        self.configure()
```

Comment:
Is that a leftover from debugging or are you proposing to ignore passed robottelo's settings and just support `settings.ini` file instead?